src/components/d2l-teacher-course-creation.js

```diff
@@ -24,17 +33,56 @@ class TeacherCourseCreation extends BaseMixin(LitElement) {
 		super();
 
 		window.tccService = TccServiceFactory.getTccService();
+
+		this.currentPage = PAGES.WELCOME_PAGE;
 	}
 
 	connectedCallback() {
 		super.connectedCallback();
 	}
 
+	_changePage(event) {
+		if (event.detail && event.detail.page) {
+			this.currentPage = event.detail.page;
+		}
+	}
+
 	render() {
-		return html`
-			<h2>Hello ${this.prop1}!</h2>
-			<div>Localization Example: ${this.localize('tccToolName')}</div>
-		`;
+		if (this.currentPage === PAGES.WELCOME_PAGE) {
+			return html `
+			<d2l-tcc-welcome
+				@change-page=${this._changePage}
+			></d2l-tcc-welcome>
+			`;
```

Comment:
nit: I prefer the ending > to not be on the same line as the ending tag. 
```suggestion
			<d2l-tcc-welcome
				@change-page=${this._changePage}>
			</d2l-tcc-welcome>
			`;
```